src/main/java/taub/gameoflife/GameFrame.java

```diff
@@ -0,0 +1,49 @@
+package taub.gameoflife;
+
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class GameFrame extends JFrame
+{
+    public GameFrame()
+    {
+        setSize(80, 400);
+        setTitle("Board");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+        JPanel buttonPanel = new JPanel();
+        JButton playButton = new JButton("Play");
+        buttonPanel.add(playButton);
+        playButton.setPreferredSize(new Dimension(100, 100));
+
+        setLayout(new BorderLayout());
+
+        int rows = 25;
+        int cols = 53;
+        GameOfLifeGrid grid = new GameOfLifeGrid(rows, cols);
+        add(grid);
+        add(buttonPanel, BorderLayout.SOUTH);
+
+        grid.changeField(18, 4);
+        grid.changeField(18, 5);
+        grid.changeField(18, 6);
+
+
+        // setup actions
+        playButton.addActionListener(new ActionListener()
+        {
+            @Override
+            public void actionPerformed(ActionEvent e)
+            {
+                grid.game.nextGeneration();
+                grid.repaint();
```

Comment:
This should go in a timer, we can discuss this tomorrow.

Reply:
You can do it now if you can figure it out.

Reply:
Ok. I will create a pr without the timer for now, but I will work on it afterwards and see if I can figure it out.